packages/ui5-config/src/types/middlewares.ts

```diff
@@ -38,6 +38,7 @@ export interface FioriToolsProxyConfig {
     backend?: FioriToolsProxyConfigBackend[];
     ui5?: Partial<FioriToolsProxyConfigUI5>;
     ignoreCertError?: boolean;
+    afterMiddleware?: string;
```

Comment:
Please remove this here again. The `afterMiddleware` property is not part of the proxy config, it is a separate property for every middleware

Reply:
Fixed in [9310a2a](https://github.com/SAP/open-ux-tools/pull/2146/commits/9310a2aec0854ff120effa183bb33a63b15e221f)